apps/admin-page/src/app/components/layout/layout.component.html

```diff
@@ -0,0 +1,68 @@
+<nz-layout>
+    <!-- Navbar -->
+    <nz-header class="navbar" data-cy="navbar">
+        <div>TC Management</div>
```

Comment:
When clicked on `TC management`, the user should be redirected to the default page (`/`)